gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceAT.java

```diff
@@ -150,6 +163,20 @@ public void getUserInProject() {
         assertThat(user, notNullValue());
     }
 
+    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "disableUserInProject")
```

Comment:
Can you please describe me this test annotation?

Reply:
```suggestion
    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "disableUserInProject", expectedExceptions = UserInProjectNotFoundException.class)
```

Reply:
you are grouping tests and then I can see in pom.xml that some groups are executed and some excluded.

Reply:
Now I see it thanks

Reply:
Well, there's some other reason of groups - some other ATs can be dependant on them, see e.g. [here](https://github.com/gooddata/gooddata-java/blob/47a6468e785b875fe6c200a21ef0b149ed9b1116/gooddata-java/src/test/java/com/gooddata/sdk/service/notification/NotificationServiceAT.java#L27).